.github/workflows/test.yaml

```diff
@@ -72,6 +72,21 @@ jobs:
           TEST_WALLET_ADDRESS: ${{ secrets.TEST_WALLET_ADDRESS }}
           FUEL_TESTNET_NETWORK_URL: ${{ secrets.FUEL_TESTNET_NETWORK_URL }}
 
+  create-fuels-e2e:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    needs: [environments]
    if: ${{ !github.event.pull_request.draft }}
    runs-on: ubuntu-latest
```

Reply:
After analyzing the code I realized that we're running these tests against a local node so it's not needed.